helm/external-secrets/values.yaml

```diff
@@ -68,11 +68,19 @@ giantswarm:
 
 replicaCount: 1
 
+# -- Specifies the amount of historic ReplicaSets k8s should keep (see https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy)
+revisionHistoryLimit: 10
+
 crds:
   # -- If true, create CRDs for Cluster External Secret.
   createClusterExternalSecret: true
   # -- If true, create CRDs for Cluster Secret Store.
   createClusterSecretStore: true
+  # -- If true, create CRDs for Push Secret.
+  createPushSecret: true
```

Comment:
I am faintly remembering this was removed back then because we do not use it. Or you can add it to the CRD as a condition I think.

Reply:
https://github.com/giantswarm/external-secrets/blob/3106466b773a3f6b4173d48751249bf7b1972fe5/helm/external-secrets/crd-base/pushsecret.yaml#L1

&& 

https://github.com/giantswarm/external-secrets/blob/3106466b773a3f6b4173d48751249bf7b1972fe5/helm/external-secrets/values.yaml#L80